src/AsyncSuffix/Workflows/RenameOverloadsPageDecorator.cs

```diff
@@ -38,20 +38,20 @@ public EitherControl View
 
         public IRefactoringPage Commit(IProgressIndicator pi)
         {
-            var irefactoringPage = OverloadsPage.Commit(pi);
-            var controlViewModel = irefactoringPage as RenameInitialControlViewModel;
-            if (controlViewModel?.View != null)
-            {
-                var renameInitialControl = controlViewModel.View.Control as RenameInitialControl;
-                var field = typeof(RenameInitialControl).GetField("myEditboxName", BindingFlags.Instance | BindingFlags.NonPublic);
-                if (field != null)
-                {
-                    EditBox = field.GetValue(renameInitialControl) as NameCompletionEdit;
-                    
-                    EditBox.GotFocus += (sender, args) => ShowSuggests();
-                }
-            }
-            return irefactoringPage;
+            var refactoringPage = OverloadsPage.Commit(pi);
+            //var controlViewModel = refactoringPage as RenameInitialControlViewModel;
```

Comment:
I wasn't able to get this part working as ReSharper has been refactored a lot internally.

Reply:
Yeah, I know, it's a bit tricky. I'll take a look one day...